java/src/main/native/src/row_conversion.cu

```diff
@@ -187,8 +189,98 @@ struct batch_data {
   std::vector<row_batch> row_batches; // information about each batch such as byte count
 };
 
-struct row_offset_functor {
-  row_offset_functor(size_type fixed_width_only_row_size)
+/**
+ * @brief functor to compute row size with alignment
+ *
+ */
+struct aligned_row_size_functor {
```

Comment:
Wait, I don't see this struct is used anywhere?

Reply:
Correct, that comes with the next step, which is #10235